### ISSUE TITLE
License File

### ISSUE THREAD

**Issue by xkortex** [Kitware/kwimage#1]
Hi Jon 👋 

I noticed in the setup.py it says the license is Apache 2.

https://github.com/Kitware/kwimage/blob/master/setup.py#L273

If that is still the case, would you mind adding a LICENSE file to kwimage, kwarray, and the like, just to be a bit more explicit? 

Thanks,

**Comment by Erotemic** [comment]
It also includes the `'License :: OSI Approved :: Apache Software License'` classifier in the setup.py, in case that helps double verify I actually meant to put the Apache 2 License in each repo. 

I'm used to have github just adding that file for me, so I must have overlooked it when I made these projects on gitlab. I'll add those files in one of my next maintence rounds. 

If you need it for whatever reason feel free to submit an MR on gitlab that adds this file: https://github.com/Erotemic/ubelt/blob/master/LICENSE to any repo where you'd like it. Otherwise, I'll get to it after I clear out my backlog.

**Comment by Erotemic** [comment]
I added this in the 0.7.9 branch, so it will be available once that is merged into master and released.

**Comment by xkortex** [comment]
Awesome, thanks!
